### PR TITLE
Michhhuang/tri 83 allow user to select attributes to measure biasfairness for

### PR DESCRIPTION
**Pull request by 1michhu1**
https://github.com/user-attachments/assets/f27d722e-89b9-4e27-8f2e-8f0d335aed73

- Create new route for uploading file
- link is currently dummy
- accessibility is still work in progress, committing rn so the s3 link can be connected 
- attribute value selection are dummy right now

### CODE REVIEW

File: frontend/src/views/pages/UploadPage/UploadPage.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                            name="bucketInput"
+                            className={styles.bucketUrlInput} 
+                            type="url"
+                            placeholder="paste public s3 bucket link" 
+                        />
+
+                        <Button 
+                            onClick={() => setPage(2)}
+                            sx={{ 
+                                backgroundColor: "#00CF31",
```
Comment (1zhaohel) [inline frontend/src/views/pages/UploadPage/UploadPage.tsx:70]: is it possible to put the stylings into a css file/create a component with the styling?

**Comment by 1zhaohel** [comment]
looks really good :D btw just looking off of the preview netlify link, should the button on main be routing to the upload page?

**Review by aerymist — APPROVED** [review]

**Review by aakshataa — APPROVED** [review]
Looks great!

**Review by aakshataa — CHANGES_REQUESTED** [review]
these are just nitpicky comments:

- increase the size of Choose File and decrease the size of Upload (to make them around the same height)
- if possible can the 'or' text and the filename be changed to use the same font as the rest of the page?
- Add footer to the upload page 

Thank You!

**Review by aakshataa — APPROVED** [review]
Looks good!!

**Comment by chloehylee** [comment]
I know this is already merged but I was just running through main again, and I was wondering if we should have some sort of indicator to show that something is selected? B/c right now after an attribute is selected the user has to manually remember which ones they chose already.

**Pull request merged**